package.json

```diff
@@ -13,24 +13,45 @@
     "url": "https://github.com/Financial-Times/n-topic-card/issues"
   },
   "homepage": "https://github.com/Financial-Times/n-topic-card#readme",
+  "scripts": {
+    "prepare": "npx snyk protect || npx snyk protect -d || true",
+    "preinstall": "[ \"$INIT_CWD\" != \"$PWD\" ] || npm_config_yes=true npx check-engine"
+  },
+  "dependencies": {
+    "@financial-times/n-myft-ui": "28.0.1",
+    "@financial-times/o-buttons": "^7.2.0",
```

Comment:
all the o- dependencies should be in peerDependencies rather than here

Reply:
Moving them.